### PR TITLE
improve etterfilter examples

### PR DESCRIPTION
**Pull request by iw4p**
add "filter engine: Cannot open file ./logfile.log" error solution.
add and edit some comments and examples for a better understanding of functions.

### CODE REVIEW

**Comment by LocutusOfBorg** [comment]
Looks good to me!

File: share/etter.filter.examples [hunk 1]
```diff
@@ -23,6 +23,14 @@ if (ip.proto == TCP) {
 }
 
 
+# log all IEC 61850 traffic when the destination ip is 192.168.1.2
+if (if (ip.dst == '192.168.1.2' && tcp.src == 102) {
```
Comment (koeppea) [inline share/etter.filter.examples:27]: Have you tested this expression? I think this is not correct and would result in failure when compiling with `etterfilter`.

Comment (iw4p) [reply share/etter.filter.examples]: It's a typo problem I wrote two if, my mistake.

**Comment by koeppea** [comment]
Thanks. Merged.

**Comment by LocutusOfBorg** [comment]
Hello, sadly this change
+replace("\x00\x00\xa2", "\x00\x01\xa2")

 resulted into an error in manpage showing this:
              replace("00\xa2", "01)

and an error in pdf generation
troff: /home/runner/work/ettercap/ettercap/build/man/etterfilter.8:276: cannot use character '0' as a starting delimiter
troff: /home/runner/work/ettercap/ettercap/build/man/etterfilter.8:276: cannot use character '0' as a starting delimiter
troff: /home/runner/work/ettercap/ettercap/build/man/etterfilter.8:276: cannot use character '0' as a starting delimiter
troff: /home/runner/work/ettercap/ettercap/build/man/etterfilter.8:276: cannot use character '0' as a starting delimiter

My fix in 40a5a8cc63f81bc3f3dd5b24f19ff6590b00b342 fixes this issue

              replace("\x00\x00\xa2", "\x00\x01\xa2")

**Pull request merged**